docs/rules/strict.md

```diff
@@ -30,18 +32,22 @@ This rule is aimed at using strict mode directives effectively, and as such, wil
 
 There are four options for this rule:
 
-* `"never"` - don't use `"use strict"` at all
-* `"global"` - require `"use strict"` in the global scope
-* `"function"` - require `"use strict"` in function scopes only
-* `"safe"` - require `"use strict"` globally when inside a module wrapper and in function scopes everywhere else.
+* `"safe"` - require `"use strict"` globally when inside a module wrapper and in function scopes everywhere else. This is the default.
+* `"never"` - don't use `"use strict"` at all.
```

Comment:
we use `disallow` as opposite of require in most of the short descriptions for the rule. Probably a good convention to stick with.